modin/data_management/factories.py

```diff
@@ -210,3 +210,43 @@ def prepare(cls):
         from modin.experimental.engines.pyarrow_on_ray.io import PyarrowOnRayIO
 
         cls.io_cls = PyarrowOnRayIO
+
+
+class ExperimentalPandasOnCloudrayFactory(ExperimentalBaseFactory):
```

Comment:
Can you move this class into `modin/experimental` module?

Reply:
I think we want all factory objects in the same place for simplicity. It allows us to go to one place for all supported combinations. Other `ExperimentalXFactory` objects live in this file as well.